features/articles/Article.js

```diff
@@ -129,7 +129,8 @@ const Article = ({
 
       <div className="wir-content-padding">
         <div className={styles['article-page-content']}>
-          {type === 'audio' && <AudioPlayer trackId={audio.id} />}
+          {/* TODO: Add Spotify and Apple to trackIds. We currently don't have it in Fibery. */}
```

Comment:
idea: maybe we can do it automatically via api:
- https://api.spotify.com/v1/shows/61Q5hmZfpw3fDoNM6cGir5

- https://developer.spotify.com/documentation/web-api/reference/get-a-show

Reply:
Maybe. Still you would need some matching logic between specific episodes uploaded and articles in Fibery. Sorting by time or matching titles could be not super reliable.